src/Administration/Resources/app/administration/src/app/init/cms.init.ts

```diff
@@ -18,7 +18,7 @@ export default function initializeCms(): void {
             previewComponent: 'sw-cms-el-preview-location-renderer',
             configComponent: 'sw-cms-el-config-location-renderer',
             appData: {
-                baseUrl: extension.baseUrl,
```

Comment:
I guess this was intended as using the same URL would make these whole sandboxing useless as you can access all the local storage, session storage and cookies of the domain and don't be forced to use the SDK

Reply:
Maybe @shopware/ct-admin knows more

Reply:
You can't access cookies and javascript/dom. It isn't enough?

Reply:
This is by design. Only with different base urls we can absolutely make sure that 2 apps are differentiated and get the right permissions.